ale_linters/elixir/credo.vim

```diff
@@ -29,9 +29,16 @@ function! ale_linters#elixir#credo#Handle(buffer, lines) abort
     return l:output
 endfunction
 
+function! ale_linters#elixir#credo#GetCommand(buffer) abort
+    let l:project_root = ale_linters#elixir#mix#FindProjectRoot(a:buffer)
```

Comment:
Move the function to `autoload/ale/handlers/elixir.vim` and update the function calls.

Reply:
@w0rp do you mean I have to move the function that find the project root on the handlers?

Reply:
Yeah, move the function to another file that all elixir linters can use, rather than using a `mix` function in the `credo` file.